lib/resorts/cypress-mountain/index.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  selector: '#conditions_lifts_1339bb0d2df6247a6d9778c66b41883b div.breakInsideAvoid',
+  parse: {
+    name: {
+      child: '1'
+    },
+    status: {
+      child: '0/0',
+      attribute: 'alt',
+      fn: status => {
```

Comment:
I removed it since 'Closed' and 'Open' are coerced automatically by liftie. See `coerce.js`. We can re-add function if more statuses need to be detected.